code/game/machinery/computer/atmos_control.dm

```diff
@@ -254,6 +197,17 @@ var/global/list/atmos_controllers = list()
 		screen = ACA_SCREEN_DETAILSVIEW //this dumb hack stops unauthorized cards from seeing shit they shouldn't
 
 	data["aca_screen"] = screen //aca_screen so we don't conflict with air alarms, which already use screen
+
+	var/list/gas_datums=list()
+	for(var/gas_id in XGM.gases)
+		var/datum/gas/gas_datum = XGM.gases[gas_id]
+		var/list/datum_data = list()
+		datum_data["id"] = gas_id
+		datum_data["name"] = gas_datum.name
+		datum_data["short_name"] = gas_datum.short_name != null ? gas_datum.short_name : gas_datum.name
```

Comment:
```suggestion
		datum_data["short_name"] = gas_datum.short_name || gas_datum.name
```
Though this isn't equivalent if `gas_datum.short_name` is an empty string instead of `null`, so ignore this if you care about that case